plyer/platforms/linux/wifi.py

```diff
@@ -1,98 +1,186 @@
 '''
-    Note::
-        This facade depends on:
-        - nmcli (Network Manager command line tool)
-            It is found in most of the popular distros. Support for other
-            managers is not provided yet.
-
-        - python-wifi module
-            `https://wifi.readthedocs.io/en/latest/`
-            `https://github.com/rockymeza/wifi`
-
+	Note::
+		This facade depends on:
+		- nmcli (Network Manager command line tool)
+			It is found in most of the popular distros. Support for other
+			managers is not provided yet.
+			Source from: https://cgit.freedesktop.org/NetworkManager/NetworkManager/tree/examples/python/gi/show-wifi-networks.py
 '''
-
+#from __future__ import print_function
 from plyer.facades import Wifi
 from subprocess import Popen, PIPE, call, STDOUT
+import gi
```

Comment:
This is a strict no, we can not just import a gui lib for managing wifi `nmcli` is not dependent on the gui lib. Think of distress that have no gnome libs at all.